client/my-sites/checkout/plan-upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,416 @@
+								{
+									components: { u: <u /> },
+								}
+							) }
+						</p>
+						<p>
+							{ translate(
+								'Are you ready to get started? Go for it now, because this special one-time offer will be gone once you leave this screen.'
+							) }
+						</p>
+
+						<p>
+							<b>
+								{ translate(
+									'Upgrade to the Premium plan (with %(bundleValue)s worth of premium themes) for just {{del}}%(fullPrice)s{{/del}} %(discountPrice)s/yr.',
+									{
+										components: { del: <del /> },
+										args: {
+											bundleValue: formatCurrency( bundleValue, currencyCode, { precision: 0 } ),
+											fullPrice: formatCurrency( planRawPrice, currencyCode ),
+											discountPrice: formatCurrency( planDiscountedRawPrice, currencyCode ),
+										},
+									}
+								) }
+							</b>
+						</p>
+					</div>
+					<div className="plan-upgrade-nudge__column-doodle">
+						<img
+							className="plan-upgrade-nudge__doodle"
+							alt=""
+							src="/calypso/images/illustrations/themes.svg"
+						/>
+					</div>
+				</div>
+			</Fragment>
+		);
+	}
+
+	footer() {
+		const { translate } = this.props;
+		return (
+			<footer className="plan-upgrade-nudge__footer">
+				<Button
+					className="plan-upgrade-nudge__decline-offer-button"
+					onClick={ this.handleClickDecline }
+				>
+					{ translate( 'No thanks, I’ll stick with the free themes' ) }
+				</Button>
+				<Button
+					primary
+					className="plan-upgrade-nudge__accept-offer-button"
+					onClick={ this.handleClickAccept }
+				>
+					{ translate( 'Yes, give me those Premium designs!' ) }
+				</Button>
+			</footer>
+		);
+	}
+
+	render() {
+		const {
+			selectedSiteId,
+			isLoading,
+			hasProductsList,
+			hasSitePlans,
+			translate,
+			receiptId,
+		} = this.props;
+		const title = translate( 'Checkout ‹ Plan Upgrade', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<Main className="plan-upgrade-nudge">
+				<PageViewTracker path="/checkout/:site/plan-upgrade-nudge/:receipt_id" title={ title } />
+				<DocumentHead title={ title } />
+				<QuerySites siteId={ selectedSiteId } />
+				{ ! hasProductsList && <QueryProductsList /> }
+				{ ! hasSitePlans && <QuerySitePlans siteId={ selectedSiteId } /> }
+
+				{ isLoading ? (
+					this.renderPlaceholders()
+				) : (
+					<>
+						{ receiptId ? (
+							<CompactCard className="plan-upgrade-nudge__card-header">
+								{ this.header() }
+							</CompactCard>
+						) : (
+							''
+						) }
+						<CompactCard className="plan-upgrade-nudge__card-body">{ this.body() }</CompactCard>
+						<CompactCard className="plan-upgrade-nudge__card-footer">{ this.footer() }</CompactCard>
+					</>
+				) }
+			</Main>
+		);
+	}
+}
+
+const trackUpsellButtonClick = buttonAction => {
+	// Track calypso_upgrade_plan_upsell_decline_button_click and calypso_upgrade_plan_upsell_accept_button_click events
```

Comment:
@niranjan-uma-shankar: I've changed the event name from the old concierge one to this one. 

First time touching Tracks events on code, so not sure if this is the normal procedure for this. Tagging you on this one just in case.

Reply:
@victorespigares Following the modular upsell code changes introduced in #33996, the tracks event name now looks like this:
For Accept: `calypso_plan_upgrade_upsell_accept_button_click`
For Decline: `calypso_plan_upgrade_upsell_decline_button_click`

Please let me know in case you want some changes to it.

Reply:
LGTM!